src/partials/benefits/benefits.html

```diff
@@ -1 +1,63 @@
-<section class="benefits">BENEFITS</section>
+<section class="benefits">
+  <div class="container">
+    <div class="benefits-wrapper">
+      <h2 class="benefits-section-title">BENEFITS OF WORKING WITH ME</h2>
+      <ul class="benefits-list">
+        <li class="benefits-items-list">
+          <div class="icon-wrapper">
+            <svg class="benefits-icon" width="24" height="24">
+              <use href="/assets/svg/icons.svg#icon-user"></use>
+            </svg>
+          </div>
+          <h3 class="benefits-list-title">Expertise</h3>
+          <p class="benefits-text">
+            As a highly experienced developer, I have deep knowledge of
+            programming and website development.
+          </p>
+        </li>
+        <li class="benefits-items-list">
+          <div class="icon-wrapper">
+            <svg class="benefits-icons" width="24" height="24">
+              <use href="/assets/svg/icons.svg#icon-message-chat-circle"></use>
+            </svg>
+          </div>
+          <h3 class="benefits-list-title">Communication</h3>
+          <p class="benefits-text">
+            Understanding your needs and wants is my priority and I am always
```

Comment:
please use this class main-text-with-opacity to <p> text please  not duplicate styles